build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+RUN_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+COMMIT_HASH=$(git rev-parse --short=10  HEAD)
+DATE_STR=$(date +"%Y%m%d%H%M%S")
+
+################################################################################
+# joshua-agent
+################################################################################
+docker build \
```

Comment:
Should we also tag this as `latest` so we can use the default agent tag in agent-scaler Dockerfile?

Reply:
added latest to the tags created by build.sh